mods/tnt/init.lua

```diff
@@ -1,11 +1,6 @@
 tnt = {}
--- Default to enabled in singleplayer and disabled in multiplayer
-local singleplayer = minetest.is_singleplayer()
-local setting = minetest.setting_getbool("enable_tnt")
-if (not singleplayer and setting ~= true) or
-		(singleplayer and setting == false) then
-	return
-end
+-- Default to enabled
+local enable_tnt = minetest.setting_getbool("enable_tnt") ~= false
```

Comment:
maybe due to t4im's comment
https://github.com/minetest/minetest_game/pull/1085#discussion_r68776441